cloudsql/sqlserver/database-sql/cloudsql.go

```diff
@@ -238,6 +239,24 @@ func initTCPConnectionPool() (*sql.DB, error) {
 
 	dbURI := fmt.Sprintf("server=%s;user id=%s;password=%s;port=%s;database=%s;", dbTCPHost, dbUser, dbPwd, dbPort, dbName)
 
+	// [START_EXCLUDE]
+	// [START cloud_sql_sqlserver_databasesql_sslcerts]
+	// (OPTIONAL) Configure SSL certificates
+	// For deployments that connect directly to a Cloud SQL instance without
+	// using the Cloud SQL Proxy, configuring SSL certificates will ensure the
+	// connection is encrypted. This step is entirely OPTIONAL.
+	dbRootCert := os.Getenv("DB_ROOT_CERT") // e.g., '/path/to/my/server-ca.pem'
+	if dbRootCert != "" {
+		// Get connection host name to be matched with host name in SSL certificate.
+		var instanceConnectionName = mustGetenv("INSTANCE_CONNECTION_NAME")
+		hostNameParts := strings.Split(instanceConnectionName, ":")
```

Comment:
Should there be an error check to ensure there are three parts here?

Reply:
Updated to add this.